tests/pldi24/test_gemmini_matmul_ae_rewrite.py

```diff
@@ -102,9 +104,9 @@ def test_matmul_ae(golden):
     res_alloc = res_load.prev()
 
     # Tile loops for a scratchpad and an accumulator
-    gemmini, _ = tile_loops_top_down(gemmini, [(i_loop, 16), (j_loop, 16)])
-    gemmini, [_, j_outer] = tile_loops_top_down(gemmini, [(i_loop, 16), (j_loop, 16)])
-    gemmini, _ = tile_loops_top_down(gemmini, [(k_loop, 16)])
+    gemmini, _ = perfect_tile(gemmini, [(i_loop, 16), (j_loop, 16)])
```

Comment:
Previously, `tile_loops_top_down` worked for both `conv` and `matmul` Gemmini schedules. Does `perfect_tile` not apply to the `conv` schedule?

Reply:
`perfect_tile` assumes that the loop is perfectly divisible even when the loop bound is not constant, but this makes me realize that we can just `try/except` that in `tile_loops_top_down` too. Maybe I'll change that